src/redturtle/prenotazioni/upgrades.py

```diff
@@ -184,3 +184,7 @@ def to_1401(context):
             for workflow_action in workflow_action_conditions:
                 if workflow_action.transition == "publish":
                     workflow_action.transition = "confirm"
+
+
+def to_1402(context):
+    update_contentrules(context)
```

Comment:
attenzione. questo in generale è rischioso: mi spiego, così' mi pare di capirer stai reinstallando tutte le contentrules del sito, se qualcuno ha cambiato, ad esempio il testo, di una contentrules tu gliela sovarscrivi.

dovresti aggiungere nell'upgrade step solo la contentrules nuova. lasciando il profiles/default così com'è e aggiungendo un nuovo profiles con solo la nuova contentrules.

questa cosa vale in generale, spesso per fare veloci si ricarica nell'upgrade step un setup completo, ma non va bene. l'upgrad deve essere più selettivo

Reply:
Fatto, grazie!